ocaml/xapi/certificates.ml

```diff
@@ -19,7 +19,7 @@ module D = Debug.Make (struct let name = "certificates" end)
 
 open D
 
-let () = Mirage_crypto_rng_unix.initialize ()
+let () = Mirage_crypto_rng_unix.initialize (module Mirage_crypto_rng.Fortuna)
```

Comment:
Should we add some debug messages around this so that we see when it is happening in xensource.log?
Just in case we get some bugs in the future where something tries to call into the RNG earlier than this.

Although are we *actually* using the RNG in XAPI? It appears not, so perhaps we don't need this one in XAPI.

AFAICT only gencert's Mirage_crypto_pk.Rsa.generate would use this, and only in the testsuite (we use an external program to generate the real one I think).
Does it work if we remove the RNG initialization from gencert and XAPI? Or do we have other functions that require a RNG?

Reply:
Just tested on a host, It's OK to remove this for both gencert and xapi.exe